src/components/Details/Details.js

```diff
@@ -68,14 +68,16 @@ const Details = ({ route }) => {
       <View style={styles.movieCardPlaceholder} testID="placeholder">
         <Image source={noImage} resizeMode="center" />
       </View>
-      <FastImage
-        testID="image-background"
-        source={{
-          uri,
-        }}
-        style={styles.movieCard}
-        resizeMode={FastImage.resizeMode.cover}
-      />
+      {uri !== '' && (
```

Comment:
What happens if uri is null? I think it will render it because it is different than `''`